src/ROUTES.js

```diff
@@ -27,20 +31,23 @@ export default {
     REPORT,
     REPORT_WITH_ID: 'r/:reportID',
     getReportRoute: reportID => `r/${reportID}`,
-    IOU_BILL_CURRENCY: 'iou/split/:reportID/currency',
-    IOU_REQUEST_CURRENCY: 'iou/request/:reportID/currency',
-    getIouRequestCurrencyRoute: reportID => `iou/request/${reportID}/currency`,
-    getIouBillCurrencyRoute: reportID => `iou/split/${reportID}/currency`,
-    IOU_REQUEST: 'iou/request/:reportID',
-    IOU_BILL: 'iou/split/:reportID',
-    getIouRequestRoute: reportID => `iou/request/${reportID}`,
-    getIouSplitRoute: reportID => `iou/split/${reportID}`,
-    IOU_SEND: 'iou/send/:reportID',
-    getIOUSendRoute: reportID => `/iou/send/${reportID}`,
-    IOU_SEND_CURRENCY: 'iou/send/:reportID/currency',
-    getIouSendCurrencyRoute: reportID => `iou/send/${reportID}/currency`,
-    IOU_DETAILS: 'iou/details',
-    IOU_DETAILS_WITH_IOU_REPORT_ID: 'iou/details/:chatReportID/:iouReportID/',
+    IOU_REQUEST,
+    IOU_BILL,
+    IOU_SEND,
+    IOU_REQUEST_WITH_REPORT_ID: `${IOU_REQUEST}/:reportID?`,
+    IOU_BILL_WITH_REPORTID: `${IOU_BILL}/:reportID?`,
+    IOU_SEND_WITH_REPORTID: `${IOU_SEND}/:reportID?`,
+    getIouRequestRoute: reportID => `${IOU_REQUEST}/${reportID}`,
+    getIouSplitRoute: reportID => `${IOU_BILL}/${reportID}`,
+    getIOUSendRoute: reportID => `${IOU_SEND}/${reportID}`,
+    IOU_BILL_CURRENCY: `${IOU_BILL}/:reportID/currency`,
+    IOU_REQUEST_CURRENCY: `${IOU_REQUEST}/:reportID/currency`,
+    IOU_SEND_CURRENCY: `${IOU_SEND}/:reportID/currency`,
+    getIouRequestCurrencyRoute: reportID => `${IOU_REQUEST}/${reportID}/currency`,
+    getIouBillCurrencyRoute: reportID => `${IOU_BILL}/${reportID}/currency`,
+    getIouSendCurrencyRoute: reportID => `${IOU_SEND}/${reportID}/currency`,
+    IOU_DETAILS,
+    IOU_DETAILS_WITH_IOU_REPORT_ID: `${IOU_DETAILS}/:chatReportID/:iouReportID/`,
```

Comment:
NAB: Have we defined a pattern for how we want to organize these routes? It seems like they're grouped by the pages they're related to. In addition, I personally think it's more clear if the getter for a route is placed right after the route itself.

Reply:
>I personally think it's more clear if the getter for a route is placed right after the route itself.

I tried to reorganize things in a way that made sense to me, but definitely prioritizing the bug fix over the engineering efficiency of having things in a certain order. 😄